appserver/ejb/ejb-container/src/main/java/com/sun/ejb/containers/EjbContainerUtilImpl.java

```diff
@@ -488,6 +488,27 @@ private ThreadPoolExecutor createThreadPoolExecutor(String poolName) {
                 ? new LinkedBlockingQueue<Runnable>(queueCapacity)
                 : new SynchronousQueue(true);
 
+        // PAYARA-405 validates attributes of the thread pool to ensure no problems      
+        if (corePoolSize < 0) {
+            _logger.log(Level.WARNING, "Core Pool Size configured to be less than 0. Resetting to 0");
+            corePoolSize = 0;
+        }
+        
+        if (maxPoolSize < 1) {
+            _logger.log(Level.WARNING, "Max Pool Size configured to be less than 1. Resetting to 1");
```

Comment:
Tricky to fail fast as you would have to force a server abort somehow and this is just creating the Executor pool. Which can happen after a deployment.